include/arch/arm/cortex_m/mpu/arm_mpu_v7m.h

```diff
@@ -53,8 +53,8 @@
 #define NORMAL_OUTER_INNER_WRITE_BACK_WRITE_READ_ALLOCATE_SHAREABLE \
 	((1 << MPU_RASR_TEX_Pos) |\
 	 MPU_RASR_C_Msk | MPU_RASR_B_Msk | MPU_RASR_S_Msk)
-#define NORMAL_OUTER_INNER_WRITE_BACK_WRITE_READ_ALLOCATE_NONSHAREABLE \
-	((1 << MPU_RASR_TEX_Pos) | MPU_RASR_C_Msk | MPU_RASR_S_Msk)
```

Comment:
Here shareable attribute is changed too.

Reply:
I disagree. First of all this change was made to the macro so that its behaviour matches its name. As noted in the commit message, this macro was unused before. Then the following patch changes the RAM attributes from `NORMAL_OUTER_INNER_NON_CACHEABLE_NON_SHAREABLE` to  `NORMAL_OUTER_INNER_WRITE_BACK_WRITE_READ_ALLOCATE_NON_SHAREABLE`.

Overall it means that the shareable attribute of the RAM region is unchanged and left as disabled.

Reply:
Ok